hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/utils/db/TestRDBStore.java

```diff
@@ -239,101 +226,84 @@ public void listTables() throws Exception {
     int count = families.size();
     // Assert that we have all the tables in the list and no more.
     for (String name : families) {
-      Assert.assertTrue(hashTable.containsKey(name));
+      Assertions.assertTrue(hashTable.containsKey(name));
       count--;
     }
-    Assert.assertEquals(0, count);
+    Assertions.assertEquals(0, count);
   }
 
   @Test
   public void testRocksDBCheckpoint() throws Exception {
-    try (RDBStore newStore =
-             new RDBStore(folder.newFolder(), options, configSet)) {
```

Comment:
Can we keep these?  I'm not sure why a new store is created, but let's clean them up in separate Jira.

Reply:
OK, I think we can close the old one before open the new one.

Reply:
I've tried before, `@TempDir` will be same in `@BeforeEach` method and the `@Test` method. That's why I changed these code in the first place.

Reply:
I see.  Then we can probably keep your original code.  Thanks.

Reply:
Seems `@TempDir` behavior is changed in [JUnit 5.8](https://junit.org/junit5/docs/5.8.0/release-notes/index.html#:~:text=%40TempDir%20can%20now,to%20per_context.)